pkg/addon/addon.go

```diff
@@ -408,6 +412,21 @@ func readDefFile(a *UIData, reader AsyncReader, readPath string) error {
 	return nil
 }
 
+// readViewFile read single view file
+func readViewFile(a *InstallPackage, reader AsyncReader, readPath string) error {
+	b, err := reader.ReadFile(readPath)
+	if err != nil {
+		return err
+	}
+	filename := path.Base(readPath)
+	switch filepath.Ext(filename) {
+	case ".cue":
+		a.Views = append(a.Views, ElementFile{Data: b, Name: filepath.Base(readPath)})
+	default:
```

Comment:
we also need to apply the yaml file to the control plane cluster if there's any.

Reply:
Do you think this yaml file example is appropriate？

```YAML
apiVersion: "v1"
kind:       "ConfigMap"
metadata: 
  name:      "service-applied-resources-view"
  namespace: {{ include "systemDefinitionNamespace" . }}
data:
  template: |
      import (
          "vela/ql"
      )
      parameter: {
          appName:    string
          appNs:      string
          name?:      string
          cluster?:   string
          clusterNs?: string
      }
      response: ql.#ListAppliedResources & {
          app: {
              name:      parameter.appName
              namespace: parameter.appNs
              filter: {
                  if parameter.cluster != _|_ {
                      cluster: parameter.cluster
                  }
                  if parameter.clusterNs != _|_ {
                      clusterNamespace: parameter.clusterNs
                  }
                  if parameter.name != _|_ {
                      components: [parameter.name]
                  }
              }
          }
      }
      if response.err == _|_ {
          status: {
              resources: response.list
          }
      }
      if response.err != _|_ {
          status: {
              error: response.err
          }
      }
```

Reply:
Do you think this yaml file example is appropriate?

```YAML
apiVersion: "v1"
kind: "ConfigMap"
metadata:
  name: "cloud-resource-view"
  namespace: "vela-system"
data:
  template: |
    import (
    "vela/ql"
    )
    
    parameter: {
      appName: string
        appNs:   string
    }
    resources: ql.#ListResourcesInApp & {
      app: {
        name:      parameter.appName
          namespace: parameter.appNs
          filter: {
            "apiVersion": "terraform.core.oam.dev/v1beta1"
              "kind":       "Configuration"
          }
          withStatus: true
      }
    }
    status: {
      if resources.err == _|_ {
        "cloud-resources": [ for i, resource in resources.list {
          resource.object
        }]
      }
      if resources.err != _|_ {
        error: resources.err
      }
    }


```

Reply:
Yes.